arm: Add missing include

caco3 opened the pull request:
Add missing include to prevent `'EINVAL' undeclared` when using `CONFIG_NULL_POINTER_EXCEPTION_DETECTION_DWT=y`

Fixes https://github.com/zephyrproject-rtos/zephyr/issues/55144

carlocaione commented:
Please fix the compliance check. No need to close and re-open a new PR. Just force-push on top of this one.

caco3 commented:
Sorry, its my first PR for this project.

I rebased and updated the commit message, but it still complains :(

microbuilder commented:
No worries, there's always a learning curve with git and working with upstream repos, and I constantly have to look things up I don't regularly use myself.

You need to remove the extra merge commit, and only push out a single commit with your changes. Try this:

```bash
$ git reset --soft HEAD~1
$ git commit --no-edit --amend
$ git push -f origin <your_branch_name>
```

caco3 commented:
Thank you very much!

It looks better now.

microbuilder commented:
> Thank you very much!
> 
> It looks better now.

Ugg, wrong instructions, that added everything from the merge conflict into this commit. 🤦 1 second.

microbuilder commented:
Sorry, my bad. To clean this up:

```bash
$ git reset --hard origin/main
```

Then use the editor to add the missing header ref back, and commit it with a sign-off via:

```
git commit -s
```

Then force push to your fork/branch again:

```bash
$ git push -f origin <branch>
```

microbuilder commented:
This one is easy, at least:

```
author email (CaCO3 <caco@ruinelli.ch>) needs to match one of the signed-off-by entries.
```

You can set name and email, then amend the email of the last commit via:

```bash
$ git config user.email "caco@ruinelli.ch"
$ git config user.name "First Last"
$ git commit --amend --reset-author
$ git push -f origin my-branch
```

caco3 commented:
Thanks, the last part was just a typo :)

caco3 committed:
arm: Add missing include

Add missing include to prevent `'EINVAL' undeclared` when
using `CONFIG_NULL_POINTER_EXCEPTION_DETECTION_DWT=y`

Signed-off-by: George Ruinelli <caco3@ruinelli.ch>
